video/cloud-client/analyze/beta_snippets.py

```diff
@@ -274,6 +288,290 @@ def track_objects(path):
     return object_annotations
 
 
+def detect_labels_streaming(path):
+    # [START video_streaming_label_detection_beta]
+    from google.cloud import videointelligence_v1p3beta1 as videointelligence
```

Comment:
Should add a commented out variable `path` if you don't include the method def.
```
// path = 'path_to_file'

Reply:
done